src/blocks/modal.css

```diff
@@ -0,0 +1,316 @@
+  opacity: 1;
+}
+
+.modal__close-btn_confirm {
+  background-image: url(../images/close-confirm.svg);
+  top: 0px;
+  right: -45px;
+  height: 40px;
+  width: 40px;
+}
+
+.modal__title {
+  font-family: poppins;
+  margin: 0 0 32px;
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 1;
+  color: #212121;
+}
+
+.modal__title_confirm{
+  font-family: Poppins;
+  font-weight: 500;
+  font-size: 24px;
+  text-align: center;
+  margin: 0;
+}
+
+.modal__label {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  margin-bottom: 32px;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 17.6px;
+  color: #212121b2;
+  font-family: poppins;
+}
+
+.modal__input {
+  padding: 16px 12px;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 1;
+  cursor: pointer;
+  color: #212121;
+  border: 1px solid #21212166;
+  border-radius: 8px;
+  max-width: 100%;
+}
+
+.modal__input:hover {
+  color: #2121214d;
+  border: 1px solid #212121;
+}
+
+.modal__input_state_error {
+  border: 1px solid #ff000099;
+}
+
+.modal__error {
+  display: block;
+  color: #ff0000b2;
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 17.6px;
+  text-align: left;
+}
+
+.modal__submit-btn {
+  font-family: poppins;
+  width: 276px;
+  padding: 18px 0;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 1;
+  color: #fcf5e5;
+  background-color: #212121;
+  border-radius: 8px;
+  cursor: pointer;
+  justify-content: center;
+}
+
+.modal__submit-btn_disabled {
+  opacity: 50%;
+}
+
+.modal__submit-btn_delete{
+  width: 156px;
+  height: 52px;
+  color: #212121;
+  background-color: #FF6565;
+  border: none;
+  font-family: Poppins;
+  font-size: 16px;
+  font-weight: 500;
+  margin: 32px 20px 32px 149px;
+}
+
+.modal__submit-btn_delete:hover{
+  background-color: #FF3333;
+}
+
+.modal__submit-btn_cancel{
+  width: 156px;
+  height: 52px;
+  color: #212121;
+  background-color: #FFFFFF;
+  font-family: Poppins;
+  font-size: 16px;
+  font-weight: 500;
+  border: 1px solid #212121
+}
+
+.modal__submit-btn_cancel:hover{
+  color: #FFFFFF;
+  background-color: #212121;
+}
+
+.modal__caption {
+  font-family: Poppins, sans-serif;
+  font-size: 20px;
+  font-weight: 500;
+  line-height: 24px;
+  text-align: left;
+  color: #ffffff;
+  background-color: #212121;
+  margin-top: 8px;
+  padding: 4px 8px;
+  border-radius: 8px;
+  display: inline-block;
+  max-width: 100%;
+  left: 420px;
+}
+
+
+@media screen and (max-width: 627px) {
+
+  .modal__container {
+     width: 300px; 
+     margin: 0 10px; 
+     padding: 12px;
+     
+  }
+
+  .modal__container_confirm {
+    height: 208px;
+    padding: 20px 0px 12px;
+    gap: 12px;
+    margin: 0 auto;
+  }
+  
+  .modal__title {
+    font-size: 20px;
+    font-family: Poppins;
+    margin-bottom: 12px;
+    justify-content: space-between;
+    line-height: 120%;
+  }
+
+  .modal__title_confirm{
+    font-size: 20px;
+    margin: 0;
+  }
+  
+  .modal__close-btn {
+   top: -40px;
+   left: 245px;
+  }
```

Comment:
* The `close icon` of the **form modals** must be **inside** the modal container on 320px (mobile) https://snipboard.io/8xQMFq.jpg . Only the image modal has it **outside**

Please, add these styles here

```css
  .modal__close-btn {
    top: 5px;
    left: 260px;
  }
```